eclass-writing/text.xml

```diff
@@ -47,11 +47,23 @@ Roughly speaking, there are three kinds of eclass:
 <title>Adding and Updating Eclasses</title>
 <body>
 
+<note>
+Given that updating eclasses will invalidate the cache of all consumer ebuilds,
+it is good etiquette to ping other developers in e.g. <c>#gentoo-dev</c> on IRC
+or informally determine if there are other similar changes (mainly documentation)
+which should be pushed at the same time in order to avoid unnecessary cache regeneration
+within a few hours or days of each other.
+
+The same applies to your own work <d/> please prepare all of your commits and
+batch push them at once where possible.
+</note>
+
 <p>
 Before committing a new eclass to the tree, it must be emailed to the gentoo-dev
-mailing list with a justification and a proposed implementation. Do not skip
+mailing list with a justification, and a proposed implementation. Do not skip
 this step <d/> sometimes a better implementation or an alternative which
-does not require a new eclass will be suggested.
+does not require a new eclass will be suggested. It might even be that a
+similar eclass has existed in the past which you weren't aware of!
```

Comment:
Does this case happen so often that it's worth mentioning? I think it can be subsumed under "better implementation" above.

Reply:
I wanted to raise the idea that it's worth consulting the past. For example, a newer developer didn't know about PROVIDES and was casually chatting with me about its benefits. They were really surprised to learn it used to be a thing!

How about we compromise and say "better (or relevant historical) implementation", which is shorter?

Reply:
But this has parentheses which should be used sparingly.

I really think that the two examples "better implementation" and "no eclass needed" are enough justification here. We don't need a complete enumeration of all possible things that could happen. IMHO it would only distract from the main message of the paragraph.

Reply:
Okay, removed.